library/std/src/f128.rs

```diff
@@ -1269,4 +1269,61 @@ impl f128 {
         let x = unsafe { cmath::lgammaf128_r(self, &mut signgamp) };
         (x, signgamp)
     }
+
+    /// Error function.
+    ///
+    /// # Unspecified precision
+    ///
+    /// The precision of this function is non-deterministic. This means it varies by platform, Rust version, and
+    /// can even differ within the same execution from one invocation to the next.
+    /// This function currently corresponds to the `erff128` from libc on Unix
+    /// and Windows. Note that this might change in the future.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// #![feature(f128)]
+    /// #![feature(float_erf)]
+    /// let x: f128 = 1.0;
+    ///
+    /// let abs_difference = (x.erf() - 0.8427007929497148693412206350826093).abs();
+    ///
+    /// assert!(abs_difference <= f128::EPSILON);
+    /// ```
```

Comment:
The `f128` examples need to be gated in

```
    /// # #[cfg(reliable_f128_math)] {
    /// ...
    /// # }
```

like the others here, since a lot of platforms don't provide those symbols yet and adding them to `libm` is still a work in progress.